content/docs/engine-dev/tools/project-manager/source-code-map.md

```diff
@@ -0,0 +1,110 @@
+{{< /note >}}
+
+{{< note >}}
+This contains brief summaries of various class's functionality and role. If available, there are additional links in the table of contents, either with expanded information, or github links.
+{{< /note >}}
+
+## Map
+- <b>Startup Classes</b>
+    - main.cpp [[cpp file](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/main.cpp) ]
+        - Entry point for the Project Manager. Initiates the [Application main loop](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/main.cpp#L28).
+    - [Application](#application)  [[header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.h) ] [[cpp file](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.cpp) ]
+        - Houses core startup code and configuration loading for Qt framework, Logging, Python Bindngs and O3DE Engine Registration. Facilitates prior housework to starting up the application main loop, and clean up when tearing down.
+    - [ProjectManagerWindow](#projectmanagerwindow)  [[header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ProjectManagerWindow.h) ] [[cpp file](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ProjectManagerWindow.cpp) ]
+        - Constructs the application's `QMainWindow` and `DownloadController`, and prepares all relevant transition screens for the Project Manager.
+    - [ScreensCtrl](#screensctrl)  [[header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreensCtrl.h) ] [[cpp file](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreensCtrl.cpp) ]
+        - Container structure for all Project Manager Screens and GUI Widgets. Also houses primary transition code for facilitating Project Manager's business logic.
+    - ScreenFactory  [[header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenFactory.h) ] [[cpp file](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenFactory.cpp) ]
+        - Helper class for ScreensCtrl that routes `ProjectManagerScreen` enums to appropriate ScreenWidget constructor, invokes that construtor, and returns an instance of a given screen for the Project Manager's business logic.
+
+
+### Application
+
+Inherits from `AzFramework::Application`.
+
+The functions of the Application class are exclusively run by `main.cpp`.
+
+The two primary functions to keep track of are `Init` and `Run`. [`TearDown`](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.cpp#L260-L272) is invoked at `Application`'s deconstruction.
+
+
+##### [`Application::Init`](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/Application.cpp#L34)
+* Sets various `QApplication` and `QCoreApplication` attributes and parameters
```

Comment:
Nit: We generally use periods at the end of every line of bulleted lists of information like this.